CHANGELOG.md

```diff
@@ -11,8 +11,10 @@ the code was deployed.
 ## [Unreleased]
 ### Added
 - Column definitions to the Heartbeat Dashboard (CU-bgjqh5).
```

Comment:
Hmmm, when was the last time you rebased off master? This should be in v1.6.0, I think?